Distribution/Server/Framework/Templating.hs

```diff
@@ -119,6 +124,18 @@ instance ToSElem PackageName       where toSElem = toSElem . display
 instance ToSElem Version           where toSElem = toSElem . display
 instance ToSElem PackageIdentifier where toSElem = toSElem . display
 
+-- | Format 'UTCTime' value as HTML.
+utcTimeTemplateVal :: String -> UTCTime -> (String, TemplateVal)
+utcTimeTemplateVal k utime = (k, TemplateVal (SBLE (stFromString utcSpanEl)))
```

Comment:
This is not the cleanest way. Preferably we should have the `XHtml` based helper in one place, and use it here.

I'm not familiar enough with code base to know where such utils belong.